src/infrastructure/organisations/api.js

```diff
@@ -160,41 +161,8 @@ const listInvitationServices = async (page, pageSize, correlationId) => {
   return callOrganisationsApi(`/invitations?page=${page}&pageSize=${pageSize}`, 'GET', undefined, correlationId);
 };
 
-const listRequests = async (page, filterStates, correlationId) => {
-  let uri = `organisations/requests?page=${page}`;
-  if (filterStates && filterStates.length > 0) {
-    filterStates.forEach((status) => {
-      uri += `&filterstatus=${status}`;
-    });
-  } else {
-    uri += `&filterstatus=0&filterstatus=2&filterstatus=3`
-  }
-  return callOrganisationsApi(uri, 'GET', undefined, correlationId);
-};
-
-const getRequestById = async (requestId, correlationId) => {
-  return callOrganisationsApi(`organisations/requests/${requestId}`, 'GET', undefined, correlationId);
-};
-
-const updateRequestById = async (requestId, status, actionedBy, actionedReason, actionedAt, correlationId) => {
```

Comment:
Hi @jovin-joy why have you removed these two?

Reply:
Hi Rhys, merge conflict confusion, I will correct it ;)